src/lightning_lite/plugins/precision/utils.py

```diff
@@ -0,0 +1,27 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+
+from lightning_lite.utilities.enums import PrecisionType
+
+
+def _fp_to_half(tensor: torch.Tensor, precision: PrecisionType) -> torch.Tensor:
```

Comment:
```suggestion
def _float_to_half(tensor: torch.Tensor, precision: PrecisionType) -> torch.Tensor:
```

Reply:
float is misleading, as there's many float types but it's usually meant for 32 bit. fp or floating_point is more generic: https://pytorch.org/docs/stable/generated/torch.is_floating_point.html